generator/sequence_transform_data.py

```diff
@@ -586,32 +586,32 @@ def create_test_rule_c_string(
     # we don't want any utf8 symbols in transformation string here
     transform_dict = {
         "\\": "\\\\",
-        WORDBREAK_CHAR: " ",
-        **{sym: char for sym, char in zip(MAGIC_CHARS, SEQ_TOKENS_ASCII)}
+        WORDBREAK_SYMBOL: " ",
+        **{sym: char for sym, char in zip(SEQ_TOKEN_SYMBOLS, SEQ_TOKENS_ASCII)}
     }
     for (i, j) in transform_dict.items():
         transform = transform.replace(i, j)
     trans_c_str = f'    "{transform}",'
-    seq_ints = [char_map[c] for c in sequence] + [0]
+    seq_ints = [symbol_map[c] for c in sequence] + [0]
     seq_int_str = ', '.join(map(uint16_to_hex, seq_ints))
     seq_c_str   = f'    (uint16_t[{len(seq_ints)}]){{ {seq_int_str} }},'
     return (seq_c_str, trans_c_str)
```

Comment:
not crucial in any way, but could've been replaced with
```py
seq_c_str = f'    (uint16_t[{len(seq_ints)}]){{ {seq_int_str} }},'
return seq_c_str, trans_c_str
```

Reply:
This PR is strictly renames. I didn't even consider any non-name changes. We can do code simplifications in another pass.